sample-apps/donet-sample-app/Controllers/AppController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using Amazon.S3;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Net.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Instrumentation;
+using System.Diagnostics.Metrics;
+
+namespace integration_test_app.Controllers
```

Comment:
Nit: The convention is to use pascalcase for namespaces. Can we add some style checker?